tests/unit/amo/reducers/test_experiments.js

```diff
@@ -0,0 +1,38 @@
+import reducer, {
+  initialState,
+  storeExperimentVariant,
+} from 'amo/reducers/experiments';
+import { fakeStoredVariant } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const state = reducer(undefined, { type: 'NONE' });
+
+      expect(state).toEqual(initialState);
+    });
+
+    it('stores an experiment variant', () => {
+      const storedVariant = fakeStoredVariant;
+
+      const state = reducer(
+        undefined,
+        storeExperimentVariant({ storedVariant }),
+      );
+
+      expect(state.storedVariant).toEqual(storedVariant);
+    });
+
+    it('can clear an experiment variant', () => {
+      const storedVariant = fakeStoredVariant;
+
+      let state = reducer(undefined, storeExperimentVariant({ storedVariant }));
+
+      expect(state.storedVariant).toEqual(storedVariant);
+
+      state = reducer(state, storeExperimentVariant({ storedVariant: null }));
```

Comment:
I suggest aliasing `storeExperimentVariant({ storedVariant: null })` to clear an experiment. Something like this is probably enough:

```js
const clearExperimentVariant = () => storeExperimentVariant({ storedVariant: null });
```

Reply:
That is a great idea, but I found that after changing the shape of the experiments state, it makes more sense to have an actual, implemented `clearExperimentVariant` action, so that's what I've done.